pkg/antctl/antctl.go

```diff
@@ -399,9 +400,14 @@ $ antctl get podmulticaststats pod -n namespace`,
 						},
 						{
 							name:      "networkpolicy",
-							usage:     "NetworkPolicy name. If present, Namespace must be provided.",
+							usage:     "NetworkPolicy name. If present, type must be provided. Namespace must be provided for non-cluster-scoped policy types.",
```

Comment:
> If present, type must be provided

So with the latest version, there is no way to enforce that?

Reply:
If accommodating for the missing policy type in the API is too cumbersome, I think it is ok to just require the policy type unconditionally at the API level. This is an Agent-specific API, which means it is hard to consume directly without executing antctl in the Agent Pod. I think it generally makes sense to strive to preserve backwards-compatibility, even for this API, but not if it means adding too much complexity (considering that again this is an Agent-specific API, only used for advanced troubleshooting and as far as we know exclusively consumed via antctl).

Reply:
In the latest version I've updated the logic so that 1. type is not strictly required, just like before 2. if a type is provided, validation on whether that type requires a namespace is performed 3. if there's ambiguity, a BadRequest will be returned